package.json

```diff
@@ -25,6 +25,8 @@
     "test:watch": "jest --config jest.test.config.js --watch",
     "playground:start": "yarn --cwd playground start",
     "playground:build": "yarn --cwd playground build",
+    "storybook:build": "yarn --cwd packages/application-components storybook:build",
```

Comment:
shouldn't these scripts be part of the `application-components` package?

Reply:
We should be able to use `yarn workspace application-components storybook:build` here? [ref](https://yarnpkg.com/lang/en/docs/cli/workspace/)